api/v2/v2.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package v2
+
+import (
+	"regexp"
+
+	"github.com/decred/dcrtime/merkle"
+)
+
+type ResultT int
+
+const (
+	APIVersion = 2
+
+	// VersionRoute defines a top-level API route for retrieving latest version
+	VersionRoute = "/version/"
+
+	// StatusRoute defines the API route for retrieving
+	// the server status.
+	StatusRoute = "/v2/status/"
+
+	// TimestampRoute defines the API route for submitting
+	// a single string digest.
+	TimestampRoute = "/v2/timestamp/" // Single digest timestamping
+
+	// TimestampsRoute defines the API route for submitting
+	// a batch of timestamps or digests.
+	TimestampsRoute = "/v2/timestamps/" // Multi digest timestamping
+
+	// VerifyRoute defines the API route for both timestamp
+	// and digest verification.
+	VerifyRoute = "/v2/verify/" // Multi verify digests
+
+	// ResultInvalid indicates the operation on the backend was invalid.
+	ResultInvalid ResultT = 0
+
+	// ResultOK indicates the operation completed successfully.
+	ResultOK ResultT = 1
+
+	// ResultExistsError indicates the digest already exists and was
+	// rejected.
+	ResultExistsError ResultT = 2
+
+	// ResultDoesntExistError indiciates the timestamp or digest does not
+	// exist.
+	ResultDoesntExistError ResultT = 3
+
+	// ResultDisabled indicates querying is disabled.
+	ResultDisabled ResultT = 4
+
+	// DefaultMainnetTimeHost indicates the default mainnet time host
+	// server.
+	DefaultMainnetTimeHost = "time.decred.org"
```

Comment:
I don't think these host defaults belong in the api spec.  They should be config defaults.

Reply:
Despite not disagreeing it's the best spot. API users need to know this constant and making them look for it in an obscure location is mean. Let's keep it here.